src/main/java/edu/ie3/datamodel/io/connectors/SqlConnector.java

```diff
@@ -156,4 +158,54 @@ public void shutdown() {
     }
     return insensitiveFieldsToAttributes;
   }
+
+  /**
+   * Enhancing the {@link IndividualTimeSeriesMetaInformation} with the name of the table containing
+   * the time series
+   */
+  public static class SqlIndividualTimeSeriesMetaInformation
```

Comment:
The class nearly has the same outline as `CsvIndividualTimeSeriesMetaInformation`. Maybe we could also merge both classes? Instead of `tableName` and `fullFilePath` something like `location`?

Reply:
I like your suggestion, however I also like having a distinction between csv and sql meta information